pm2.config.js

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const PATH = process.env.PATH.split(':')
+  .filter((p) => !p.includes(process.env.TMPDIR))
+  .join(':');
+
+module.exports = {
+  apps: [
+    {
+      name: 'payments-next',
+      script: 'nx serve payments-next',
+      cwd: __dirname,
+      max_restarts: '1',
+      min_uptime: '2m',
+      env: {
+        PATH,
```

Comment:
Could we change the PORT? I don't _think_ we have a convention, so maybe lets keep it close to the current payments frontend. How about `PORT: '3035'`?